AnnoyancesFilter/sections/popups.txt

```diff
@@ -105,6 +105,7 @@ nationalfm.ro###box-fb-like
 kizlarsoruyor.com###contentpopup-container
 eva.ru###subscribe-dialog
 navalny.com###pushPopup
+metrolyrics.com###videoContainer
```

Comment:
Please limit rule to mobile devices only. The video is autostarting on Windows Chrome and when it is hidden, the user has no way to pause the video.